man/repart.d.xml

```diff
@@ -217,6 +217,15 @@
         automatically used.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>Uuid=</varname></term>
+
+        <listitem><para>The UUID to assign to the partition if none is assigned yet. Note that this
+        setting is not used for matching. It is also not used when a UUID is already set for an existing
+        partition. It is thus only used when a partition is newly created. If not specified a UUID derived
+        from the partition type is automatically used.</para></listitem>
```

Comment:
hmm, i think we should apply the configured UUID in two cases: when we create a partition and when we find a matching partition where the UUID is currently unset (i.e. all zeroes).

Reply:
I updated the documentation: Looking at the code the UUID is applied either when the partition does not exist or it contains an all-zero UUID.